js/plates-model/model-worker.js

```diff
@@ -104,7 +104,6 @@ onmessage = function modelWorkerMsgHandler (event) {
     let storedModel = labeledSnapshots[data.label]
     if (storedModel) {
       self.m = model = Model.deserialize(storedModel)
-      delete labeledSnapshots[data.label]
```

Comment:
It doesn't seem necessary and it was causing bugs in authoring mode (when I wanted to go back and forth a few times).